src/frontends/jax/src/node_context.cpp

```diff
@@ -172,6 +172,12 @@ CoordinateDiff NodeContext::const_named_param<CoordinateDiff>(const std::string&
         return {};
 }
 
+template <>
+ov::element::Type NodeContext::const_named_param<ov::element::Type>(const std::string& name) const {
+    auto c = get_constant_from_params(*this, name);
+    return c->get_element_type();
+}
+
```

Comment:
I think this is not correct way how we should retrieve output type for `iota` operation. That is because it tries to create a constant that saves a value equal to type.
We need to have a different mechanism to parse and transfer such attribute values storing type.